.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
-**Change log description**
+**Change log description (2-3 concise points about the changes here for the merge log)**
 
-**Purpose of the change**
+**Purpose of the change (_e.g._, Fixes #666, Closes #1234)**
 
-**What the code does**
+**What the code does (detailed description of the code changes)**
```

Comment:
I usually prefer to merge this into the description section, since I'd like that whole blurb to be associated with the commit (when I merge a PR, I copy the description into the squash commit description).

I don't mind this clarification, but I think it may be nice to hint at which of these sections will be part of the PR merge commit description.

Reply:
> I usually prefer to merge this into the description section, since I'd like that whole blurb to be associated with the commit (when I merge a PR, I copy the description into the squash commit description).

Let's not change it to the way you are describing. The commit message should be concise and contain only 2-3 main points about the the change, otherwise the commit log will end up being unnecessarily long.